workshops/views.py

```diff
@@ -595,6 +627,12 @@ class PersonPermissions(LoginRequiredMixin, UpdateViewContext):
 def person_password(request, person_id):
     user = get_object_or_404(Person, pk=person_id)
 
+    # Either the user requests change of their own password, or someone with
+    # permission for changing person does.
+    if not ((request.user == user) or
```

Comment:
Found this a bit difficult to parse, but can't think of a simpler way to write it.